fifteen_min/src/isochrone.rs

```diff
@@ -66,9 +71,12 @@ impl Isochrone {
         start: Vec<BuildingID>,
         options: Options,
     ) -> Isochrone {
-        let time_to_reach_building = options
+        let spot_starts = start
```

Comment:
Makes total sense! Thanks for the explanation here, starting to get the hang of when to borrow and when not to. To be honest, I have been just trying to make the warnings go away so far, will start to be more thoughtful about it now.

Reply:
The only thing is start is used further down in the function so I think I need to clone it?

Reply:
Ah you're right, we do store it later. In that case, I think `let spot_starts = start.clone().into_iter().map(Spot::Building).collect();` should do the trick, unless clippy recommends otherwise